libs/api-catalogue/services/src/lib/provider.service.ts

```diff
@@ -46,6 +46,7 @@ export class ProviderService {
           (item: XroadIdentifier): Provider => {
             return {
               type: providerType,
+              name: item.name ?? '',
```

Comment:
Maybe we should have the default name behavior happening here instead of in L64 in `restmetadata.service.ts`
As here we are responsible to set the provider, and each provider is classified as the subsystem in question.